test/test-applications/integrations/TestApplication.ContinuousProfiler.ContextTracking/Program.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Diagnostics;
+
+namespace TestApplication.ContinuousProfiler.ContextTracking;
+
+internal class Program
+{
+    private static readonly ActivitySource Source = new ActivitySource("TestApplication.ContinuousProfiler.ContextTracking");
+
+    public static async Task Main(string[] args)
+    {
+        using var activity = Source.StartActivity();
```

Comment:
The Activity ends when the program exits. Is this by design, or is there a need to stop it manually after calling `DoSomethingAsync`

Reply:
This is by design.
The idea was to start an activity that remains active until async operation completes, and to verify that trace context flows properly between threads that carry out parts of the async operation.

Reply:
I think it would be good to add a comment explaining this in the source code (in this file).